suites/pacific/rgw/tier-3_rgw_multisite_archive_with_haproxy.yaml

```diff
@@ -450,7 +450,7 @@ tests:
           config:
             set-env: true
             script-name: test_bucket_lifecycle_object_expiration_transition.py
-            config-file-name: test_lc_rule_prefix_non_current_days.yaml
```

Comment:
@viduship , do we have this conf yaml in pipeline?

Reply:
@mkasturi18 it is there but not from pri to sec, we do nt have any lc tests from pri to sec in automation.

Reply:
@viduship , then IMO,
will keep this existing testcase and add new testcase for haproxy one,
do we have separate polarian ?

Reply:
haproxy should not be a separate test case but rather an ideal configuration going forward.

also, this is a generic test case of testing LC at multisite not related to any zone. Do we want to have separate tests is the question

Reply:
We have one test-case related to LC expiration in Multisite suite,
tier-1-extn_rgw_multisite-secondary-to-primary.yaml
test_lc_rule_prefix_and_tag.yaml

here since suite itself is created for haproxy one, I think we can go ahead with current changes.

Reply:
@ckulal thanks.

Reply:
Ok for me